### ISSUE TITLE
[a11y]: Screen reader announces checkmark icon as blank

### ISSUE THREAD

**Issue by jack-cameron16** [carbon-design-system/carbon#20049]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.68.0

### React version

16.12.0

### Automated testing tool and ruleset

AVT tester using a screen reader 

### Description

Issue described here: https://github.ibm.com/orgs/ibmcloud/projects/53/views/8?pane=issue&itemId=854017&issue=ibmcloud%7CStatus-and-Notifications%7C74

### Reproduction/example

refer to this issue: (https://github.ibm.com/orgs/ibmcloud/projects/53/views/8?pane=issue&itemId=854017&issue=ibmcloud%7CStatus-and-Notifications%7C74) for a screenshot

### Steps to reproduce

Navigate to the status/history page on cloud.ibm.com and use the screen reader on the table. In the transcript you'll see Column 2 followed by blank, despite their being a checkmark icon there

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by alisonjoseph** [comment]
Can you reproduce this in a [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite?file=src%2Findex.scss), we don't have access to your repo above.

**Comment by 2nikhiltom** [comment]
@jack-cameron16 could you reproduce this in a [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/examples/vite?file=src%2Findex.scss),

**Comment by jack-cameron16** [comment]
https://stackblitz.com/edit/github-gnpspddy?file=src%2FApp.jsx Just recreated here. The issue the avt tester claimed was that the screen reader does not read aloud a description of the CheckmarkFilled carbon-icon when in a DataTable. I used apple's VoiceOver when testing

**Comment by 2nikhiltom** [comment]
<img width="880" height="284" alt="Image" src="https://github.com/user-attachments/assets/23d6af49-0b07-4af5-9c14-d6659209075d" />
Hey @jack-cameron16 - This is just an icon/SVG and is not interactive or focusable, so it’s not announced by the screen reader. I also confirmed that the A11y checker isn’t flagging any violations in the StackBlitz you shared.

**Comment by 2nikhiltom** [comment]
Hi @jack-cameron16 - closing this as can not recreate. Happy to re-open if you have queries
